server/services/seed.ts

```diff
@@ -0,0 +1,32 @@
+import { Config } from '../config';
```

Comment:
This logic is tied to the Database provider so should we move it to `shared/database/seed`?